### PR TITLE
[Bluetooth] Add new API to destroy Gatt server

### PR DESCRIPTION
**Pull request by Wootak**
Add new API to destroy Gatt server

**API Changes**
- ACR: http://suprem.sec.samsung.net/jira/browse/TCSACR-216

### CODE REVIEW

**Comment by Wootak** [comment]
> Why don't you use dispose pattern?

Because all of classes of BluetoothGatt has not used IDispoable interface, I added a new method.
Is using IDisposable interface mandatory?

**Comment by myroot** [comment]
Why don't you implement IDisposable interface on BluetoothGattServer class?

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGatt.cs [hunk 1]
```diff
@@ -200,6 +201,26 @@ internal bool IsValid()
         {
             return _impl.GetHandle().IsInvalid == false;
         }
+
+        ~BluetoothGattServer()
```
[the remark this answers is not in the document]
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGatt.cs]: Done.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGatt.cs [hunk 2]
```diff
@@ -200,6 +201,26 @@ internal bool IsValid()
...
+        {
+            Dispose(false);
+        }
+
+        public void Dispose()
```
[the remark this answers is not in the document]
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGatt.cs]: Done.

**Pull request merged**